sdk/script/acquisition-sdk.ts

```diff
@@ -134,25 +148,37 @@ export class AcquisitionManager {
         });
     }
 
-    public reportStatus(status: string, message?: string, callback?: Callback<void>): void {
-        var url: string = this._serverUrl + "reportStatus";
-        var body: string;
-
-        switch (status) {
-            case AcquisitionStatus.DeploymentSucceeded:
-            case AcquisitionStatus.DeploymentFailed:
-                url += "/deploy";
-                body = JSON.stringify({ deploymentKey: this._deploymentKey, status: status, message: message });
-                break;
-
-            default:
-                if (callback) {
-                    callback(new Error("Unrecognized status."), /*not used*/ null);
-                }
-                return;
+    public reportStatus(package?: Package, status?: string, callback?: Callback<void>): void {
```

Comment:
He has AcquisitionStatus exported above, which is equivalent to an enum (minus type-checking I guess, but at least for the REST API interface I feel like string constants are a better choice than TypeScript enums, which are integer constants).

We could make this function private, and then introduce public reportSuccess() and reportFailure() functions that call this internally?